Simplenote/src/androidTest/java/com/automattic/simplenote/SimplenoteAppRunner.kt

```diff
@@ -0,0 +1,12 @@
+package com.automattic.simplenote
+
+import android.app.Application
+import android.content.Context
+import androidx.test.runner.AndroidJUnitRunner
+
+class SimplenoteAppRunner : AndroidJUnitRunner() {
+    @Throws(InstantiationException::class, IllegalAccessException::class, ClassNotFoundException::class)
```

Comment:
Question (❓): Why did you add the `@Throws` annotation here, is that required somehow?

Reply:
I do not think it is needed. I will remove it.

Reply:
Many thanks! 🙏